_inc/header.php

```diff
@@ -1,27 +1,15 @@
+			</svg>
+		</div>
 
-				<ul class="main-nav">
-					<li class="jetpack-logo"><a href="<?php echo Jetpack::admin_url(); ?>" title="<?php esc_attr_e( 'Jetpack', 'jetpack' ); ?>" <?php if ( 'jetpack' == $current ) { echo 'class="current"'; } ?>><span><?php esc_html_e( 'Jetpack', 'jetpack' ); ?></span></a></li>
-					<?php if ( ( Jetpack::is_active() || Jetpack::is_development_mode() ) && current_user_can( 'jetpack_manage_modules' ) ) : ?>
-						<li class="jetpack-modules">
-							<a href="<?php echo Jetpack::admin_url( 'page=jetpack_modules' ); ?>" class="jp-button--settings <?php if ( 'jetpack_modules' == $current ) { echo 'current'; } ?>"><?php esc_html_e( 'Settings', 'jetpack' ); ?></a>
-						</li>
-						<li class="jetpack-modules">
-							<a href="http://jetpack.com/survey/?rel=<?php echo JETPACK__VERSION; ?>" target="_blank" class="jp-button--settings"><?php esc_html_e( 'Feedback', 'jetpack' ); ?></a>
-						</li>
-					<?php endif; // End if connected or dev mode and is admin ?>
-
-					<?php if ( Jetpack::is_active() && ! Jetpack::is_development_mode() ) : ?>
-						<li class="jetpack-modules">
-							<a href="<?php echo Jetpack::admin_url( 'page=my_jetpack' ); ?>" class="jp-button--settings <?php if ( 'my_jetpack' == $current ) { echo 'current'; } ?>"><?php esc_html_e( 'My Jetpack', 'jetpack' ); ?></a>
-						</li>
-					<?php endif; ?>
-				</ul>
-
-			</nav>
-		</div><!-- .header -->
-		<div class="wrapper">
+		<ul class="jp-masthead__links">
+			<li><a href="http://jetpack.com/support/" target="_blank" class="jp-masthead__link"><span class="dashicons dashicons-editor-help" title="Need Help?"></span><span>Need Help?</span></a></li>
```

Comment:
@jeherve you should know going forward that currently nothing in this admin is translated.  We are waiting on #3805 to land